app.js

```diff
@@ -1,3 +1,5 @@
+//import { setInterval } from "core-js/library/web/timers";
```

Comment:
What?

Reply:
That was causing issues on my end. Blame my harddrive. Too scared to update NODE cos low storage.